metatron-app/metatron-discovery
# fix dashboard error

### Description
1. [Dashboard creation] Deleted without confirmation pop-up even though it is a data source related to other data sources
2. [Create Dashboard] Mandatory filter setting popup is displayed even though the mandatory filter does not exist and progress is not possible.
3. [Dashboard] Chart creation pop-up> Analyze guide phrase error and enable on / off setting button not selected
4. [Dashboard] Data source details> Column list view> Error toast is exposed when selecting column whose role is Measure when there is recommendation filter
5. [Dashboard] Dashboard view mode> left panel> select Comment> Write comment> Date display is automatically updated to the current time
6. [Create Dashboard] Manage Scheme> Join popup> Preview results> Result data summary is not the same as screen design
7. [Dashboard] Dashboard Edit Mode> Data Sources panel> Link Source icon for data source is not exposed
8. [Dashboard] dashboard view mode> main area> global filter icon is not exposed
9. [Dashboard creation] Data source related tab> Data preview (default)> Manage Scheme requires two clicks to switch views
10. [Dashboard] Dashboard edit mode> Data source panel> Dimension list> Filter icon On is selected, the index of the filter is not highlighted for 1.5 seconds
11. [Dashboard] Dashboard edit mode> Data source panel> List not collapsed when clicking the index bar
12. [Dashboard] Cursor shape is exposed as a four-way cursor when resizing through dashboard edit mode> widget placement interaction> clearance between widgets

**Related Issue** : 
METATRON-2840
METATRON-2843
METATRON-3010
METATRON-3014
METATRON-2884
METATRON-2850
METATRON-2938
METATRON-2941
METATRON-2984
METATRON-2915
METATRON-2906
METATRON-2923

### How Has This Been Tested?
1. Verify that a confirmation pop-up appears when deleting a data source that is associated with another data source when creating a dashboard.
2. If there are no required filters when creating the dashboard, verify that the Required Filter Settings pop-up does not appear and proceed.
3. When creating the chart, check that the guide phrase in the predictive line menu is displayed normally.
4. On the dashboard, select Data Source Preview> Column List View> Recommendation Filter to see if the information is displayed properly when selecting a column whose role is Measure.
5. In the dashboard view mode, check if the comment input time of the left panel is displayed normally.
6. When creating the dashboard, in the Join popup, verify that the result preview is the same as the screen design.
7. In the Dashboard Edit Data Source panel, verify that the connection key information for the data source is displayed.
8. Verify that the global filter icon is displayed in the dashboard view.
9. When creating the dashboard, verify that clicking once on the tab in the data source preview changes.
10. Check that the filter is highlighted for 1.5 seconds when you select Dashboard Edit Mode> Data Sources panel> Dimension List> Filter Icon On.
11. Check that the list collapses when you click Dashboard Edit Mode> Data Sources panel> Index Bar.
12. Dashboard Edit Mode> Widget Placement Interaction> Make sure that the cursor shape appears properly when resizing between widgets.

#### Need additional checks?


### Types of changes



### Checklist:


### Additional Context
